### PR TITLE
3-3 , 3-4 , 3-5 done

### PR DESCRIPTION
**Pull request by Cerealmaster0621**
### 3-3 get a list of items 

Struct `Items` and `Item` for json parsing

Added reusable function `readItemsFromFile` that returns current items array

### 3-4 add the list of item , image

 1. Fetch `Items` using `readItemsFromFile` function, store new Item in variable
 2. append `newItem` in `Items` array, marshal item to valid Json
 3. rewrite items.json with updated items

### 3-5 return item details 

Basically parse the input param as integer and **indexing `items`** is the idea.
 
 

## CHECKS :warning:

Please make sure you are aware of the following.

- [ ] **The changes in this PR doesn't have private information

### CODE REVIEW

**Comment by kimichael0** [comment]
The branch name should represent the your intended "unit" of work. I would call it like something related to "step-3"
